miden/src/examples/mod.rs

```diff
@@ -50,11 +51,17 @@ pub struct ExampleOptions {
 //#[clap(about = "available examples")]
 pub enum ExampleType {
     /// Compute a Fibonacci sequence of the specified length
-    Fib {
+    Fibonacci {
         /// Length of Fibonacci sequence
         #[clap(short = 'n', default_value = "1024")]
         sequence_length: usize,
     },
+
+    Blake3 {
+        /// Length of recursive hash sequence
+        #[clap(short = 'n', default_value = "32")]
+        sequence_length: usize,
```

Comment:
nit: I'd change this to `chain_length` and would also change the description to "Length of the hash chain".